src/CMakeLists.txt

```diff
@@ -40,11 +40,13 @@ set(SRC
    ngraph_cluster_manager.cc
    ngraph_deassign_clusters.cc
    ngraph_encapsulate_clusters.cc
+   ops/ngraph_encapsulate_op.cc
    ngraph_encapsulate_op.cc
    ngraph_freshness_tracker.cc
    ngraph_mark_for_clustering.cc
    ngraph_rewrite_for_tracking.cc
    ngraph_rewrite_pass.cc
+   ops/ngraph_tracked_variable.cc
```

Comment:
Also do we need to update this as well? https://github.com/tensorflow/ngraph-bridge/blob/master/bazel/BUILD#L18

Reply:
Yes